cpp/tests/constraints.cpp

```diff
@@ -15,9 +40,12 @@ TEST(Constraints, LinearJointVelocity) {
   EXPECT_EQ(ljv.nbConstraints(), 0);
   EXPECT_EQ(ljv.nbVariables(), 0);
 
-  GeometricPath path;
   int N = 10;
-  Vector gridpoints (N+1);
+  Vector gridpoints;
+  {
+    Bound I (path.pathInterval());
+    gridpoints = toppra::Vector::LinSpaced (N, I[0], I[1]);
+  }
```

Comment:
Comment for L53: The allocateParams call looks redundant in normal use. What is the rationale for separating the two steps? Is it performance-related?

Reply:
Purely historical. I don't have strong arguments for this.